apps/admin/backend/src/app.ts

```diff
@@ -292,25 +297,30 @@ function buildApi({
 
     async addCastVoteRecordFile(input: {
       path: string;
-    }): Promise<AddCastVoteRecordFileResult> {
+    }): Promise<
+      Result<
+        Admin.CvrFileImportInfo,
+        AddCastVoteRecordReportError & { message: string }
```

Comment:
@jonahkagan I know that we discussed moving all error message creating and handling to the frontend, but since I wanted to use the same message for logging (which happens on the backend) I just kept the message creation on the backend.